README.rst

```diff
@@ -69,7 +69,7 @@ If you are using multiple Travis factors,
 then you can use those factors to decide what will run.
 For example, see the following ``.travis.yml`` and ``tox.ini``:
 
-.. code-block::
+.. code-block:: yaml
```

Comment:
Done a quick search and `yaml` seems supported: 
result: https://pypi.python.org/pypi/pantable
source: https://raw.githubusercontent.com/ickc/pantable/master/README.rst